packages/esm-service-queues-app/src/patient-search/patient-scheduled-visits.component.tsx

```diff
@@ -132,7 +129,7 @@ const ScheduledVisits: React.FC<{
                 addQueueEntry(
                   response.data.uuid,
                   patientId,
-                  priority ? priority : defaultPriority,
```

Comment:
It does not make sense to default to a priority here, as the only way this form is submitted is by clicking on the priority content switcher.